metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -0,0 +1,42 @@
+namespace com.linkedin.dashboard
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.ChartUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a dashboard
+ */
+record DashboardInfo {
```

Comment:
should all fields be optional here?

Reply:
Why do you think so? I guess `title` and (possibly `description`) will always be there when you create a dashboard. If not, they will probably be empty strings. I can update them with a default of empty string, I guess. Do you think it would serve the same purpose?